initializr/src/main/resources/templates/home.html

```diff
@@ -11,174 +11,167 @@
 </head>
 <body>
 <div class="container-fluid">
-
     <div class="row">
-        <div class="initializr-header">
+        <div class="start-header">
             <div class="container">
-                <h1>Spring Initializr</h1>
-                <p>Bootstrap your application now</p>
+                <h1>start.spring.io</h1>
+
+                <p>Generate your Spring Boot project in seconds!</p>
```

Comment:
Better tagline? I don't think start.spring.io fits on the "generators category" per se.